Mathlib/Analysis/Normed/Module/Dual.lean

```diff
@@ -234,6 +239,21 @@ theorem isBounded_polar_of_mem_nhds_zero {s : Set E} (s_nhd : s ∈ 𝓝 (0 : E)
     (((dualPairing 𝕜 E).flip.polar_antitone r_ball).trans <|
       polar_ball_subset_closedBall_div ha r_pos)
 
+theorem polar_singleton {a : E} : polar 𝕜 {a} = { x | ‖x a‖ ≤ 1 } := le_antisymm
```

Comment:
Should this be `@[simp]`? I'm not sure, so feel free to argue if you disagree. If it were, then `mem_polar_singleton` is provable `by simp`.

Reply:
On balance, I think so, but I may not be the best judge of what makes a good simp lemma.